src/main/java/nz/ac/waikato/its/dspace/app/xmlui/aspect/featuredcontent/FeaturedContentForm.java

```diff
@@ -0,0 +1,117 @@
+    private static final Message T_no_current = message("xmlui.aspect.FeaturedContent.FeaturedContentForm.no_current");
+    private static final Message T_img_location = message("xmlui.aspect.FeaturedContent.FeaturedContentForm.form_img_location");
+    private static final Message T_link_target = message("xmlui.aspect.FeaturedContent.FeaturedContentForm.form_link_target");
+    private static final Message T_caption = message("xmlui.aspect.FeaturedContent.FeaturedContentForm.form_caption");
+    private static final Message T_img_location_desc = message("xmlui.aspect.FeaturedContent.FeaturedContentForm.form_img_location_desc");
+    private static final Message T_link_target_desc = message("xmlui.aspect.FeaturedContent.FeaturedContentForm.form_link_target_desc");
+    private static final Message T_set_featured_content = message("xmlui.aspect.FeaturedContent.FeaturedContentForm.set_featured_content");
+
+    @Override
+    public void addPageMeta(PageMeta pageMeta) throws SAXException, WingException, SQLException, IOException, AuthorizeException {
+        pageMeta.addTrailLink(contextPath + "/", T_dspace_home);
+        pageMeta.addTrail().addContent(T_featuredcontent_trail);
+        pageMeta.addMetadata("title").addContent(T_featuredcontent_title);
+    }
+
+    @Override
+    public void addBody(Body body) throws SAXException, WingException, SQLException, IOException, AuthorizeException {
+
+        String img_location = "";
+        String link_target = "";
+        String caption = "";
+
+        // load http-parameters to restore the input data for 'cancel' button
+        String img_location_param = parameters.getParameter("img_location",null);
+        String link_target_param = parameters.getParameter("link_target",null);
+        String caption_param = parameters.getParameter("caption",null);
+
+        String errorString = parameters.getParameter("errors",null);
+
+        ArrayList<String> errors = new ArrayList<>();
+        if (StringUtils.isNotEmpty(errorString))  {
+            Collections.addAll(errors, errorString.split(","));
+        }
+
+        Division div = body.addDivision("admin-featured-content", "primary administrative featured-content");
+        div.setHead(T_head1);
+
+        // DIVISION: current item preview
+        Division current = div.addDivision("featured-content-home", "secondary");
+        current.setHead(T_head_current);
+
+        HashMap<String, String> currentItemMap = FeaturedContentController.getFeaturedItem();
+        if( currentItemMap != null) {
+              img_location = currentItemMap.get("img_location");
+              link_target = currentItemMap.get("link_target");
+              caption = currentItemMap.get("caption");
+
+            Para currentPara = current.addPara();
+            Figure figureItem = currentPara.addFigure(img_location, link_target, caption, List.TYPE_SIMPLE);
+
+            figureItem.addContent(caption);
```

Comment:
Is this necessary? You've already set the caption in the previous line. (Fair enough if it doesn't work without setting the content as well.)